libs/langgraph/langgraph/types.py

```diff
@@ -130,7 +130,7 @@ class Interrupt:
     value: Any
     resumable: bool = False
     ns: Optional[Sequence[str]] = None
-    when: Literal["during"] = "during"
+    when: Literal["during"] = dataclasses.field(default="during", repr=False)
```

Comment:
ooc: Why are we hiding from the repr?

Reply:
Because this field shouldn't exist, it's a constant, I just can't remove it bc backwards compat